src/instances/RaidInstance.ts

```diff
@@ -2447,6 +2448,55 @@ export class RaidInstance {
         });
     }
 
+    public static async compileDeadFeedbackHistory(feedbackChannel: TextChannel, storageChannel: TextChannel) {
```

Comment:
Add JSDocs explaining what this method does. Also specify return type of `Promise<void>`.